tests/test_pandas.py

```diff
@@ -1973,3 +1973,9 @@ def g(x: pd.Series) -> int:
         ),
         pd.DataFrame,
     )
+
+
+def test_tolist() -> None:
+    data = {"Courses": "pandas", "Fees": 20000, "Duration": "30days"}
+    s = pd.Series(data)
+    check(assert_type(s.array.tolist(), list), list)
```

Comment:
Can you add a test for the original issue:  `pd.array([1,2,3]).tolist()`

Reply:
Done

Reply:
You didn't do what I asked.  I should see this snippet in the test:
```python
pd.array([1,2,3]).tolist()
```

Reply:
actually we have to give it some positional argument

Reply:
You still have not added the test  `pd.array([1,2,3]).tolist()`.  The test you had before was correct when you wrote `s.array().tolist()`.  But you need to do BOTH tests.

Reply:
`check(assert_type(pd.array([1, 2, 3]).tolist(), list), list)` i have added thhis test but with warning as we need a positional argument but in this ex there is no positional argument

Reply:
> Any idea how we should change the func so that the test runs

The issue here is that `pd.array()` is returning the wrong type.

If you go into `pandas-stubs\core\construction.pyi`, and change the return type of `array()` from `ABCExtensionArray` to `ExtensionArray` (and fix the imports, etc.), then that should fix the problem.

Reply:
done